### PR TITLE
rename all external naming/config to aggregator

### PR DESCRIPTION
**Pull request by ameijer**

### CODE REVIEW

File: cost-analyzer/templates/_helpers.tpl [hunk 1]
```diff
@@ -212,9 +212,9 @@ app: query-service
 {{ include "kubecost.federator.chartLabels" . }}
 app: federator
 {{- end -}}
-{{- define "waterfowl.commonLabels" -}}
-{{ include "kubecost.waterfowl.chartLabels" . }}
-app: waterfowl
+{{- define "aggregator.commonLabels" -}}
+{{ include "kubecost.aggregator.chartLabels" . }}
+app: kubecost-query-handler
```
Comment (michaelmdresser) [inline cost-analyzer/templates/_helpers.tpl]: 👀

File: cost-analyzer/templates/_helpers.tpl [hunk 2]
```diff
@@ -239,10 +239,10 @@ app.kubernetes.io/name: {{ include "federator.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 app: federator
 {{- end -}}
-{{- define "waterfowl.selectorLabels" -}}
-app.kubernetes.io/name: {{ include "waterfowl.name" . }}
+{{- define "aggregator.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "aggregator.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
-app: waterfowl
+app: kubecost-query-handler
```
Comment (michaelmdresser) [inline cost-analyzer/templates/_helpers.tpl]: 👀

File: cost-analyzer/templates/queryhandler-cloud-cost-deployment-template.yaml [hunk 3]
```diff
@@ -116,24 +116,24 @@ spec:
...
-      {{- if .Values.waterfowl.priority.enabled }}
-      {{- if .Values.waterfowl.priority.name }}
-      priorityClassName: {{ .Values.waterfowl.priority.name }}
+      {{- if .Values.kubecostAggregator.priority }}
+      {{- if .Values.kubecostAggregator.priority.enabled }}
+      {{- if .Values.kubecostAggregator.priority.name }}
+      priorityClassName: {{ .Values.kubecostAggregator.priority.name }}
       {{- else }}
-      priorityClassName: {{ template "cost-analyzer.fullname" . }}-waterfowl-priority
+      priorityClassName: {{ template "cost-analyzer.fullname" . }}-query-handler-priority
```
Comment (michaelmdresser) [inline cost-analyzer/templates/queryhandler-cloud-cost-deployment-template.yaml]: 👀

File: cost-analyzer/templates/_helpers.tpl (hunk 1 above)
Comment (ameijer) [reply cost-analyzer/templates/_helpers.tpl]: one rename too far... thank you for catching this 🙈

**Review by michaelmdresser — APPROVED** [review]

**Pull request merged**